# AwaitableEventGroup&lt;T&gt;.TrySetException method

Attempts to transition the underlying task into the Faulted state and binds it to the specified exception.

```csharp
public virtual bool TrySetException(Exception exception)
```

| parameter | description |
| --- | --- |
| exception | The exception to bind to this task. |

## Return Value

True if the `AwaitableEventGroup` was successful; otherwise, false.

## See Also

* class [AwaitableEventGroup&lt;T&gt;](../AwaitableEventGroup-1.md)
* namespace [Microsoft.Coyote.Actors](../AwaitableEventGroup-1.md)
* assembly [Microsoft.Coyote.Actors](../../Microsoft.Coyote.Actors.md)

<!-- DO NOT EDIT: generated by xmldocmd for Microsoft.Coyote.Actors.dll -->
